hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -117,12 +117,46 @@
 
     [Install]
     WantedBy=multi-user.target
+- path: /tmp/devstack-post.sh
+  permissions: 0755
+  content: |
+    #!/bin/bash
+
+    set -o -x
+
+    # Add environment variables for auth/endpoints
+    echo 'source /opt/stack/devstack/openrc admin admin' >> /opt/stack/.bashrc
+
+    source /opt/stack/devstack/openrc admin admin
```

Comment:
a separate question

do we have any places that we use `demo` which is non-admin user and tenant to do some test? 
I knew it's kind of overkill but just want to see whether we already do it or potentially can do it